rules/prometheus-rules.yaml

```diff
@@ -1707,17 +1729,57 @@ spec:
         ) * 100 > 10
       for: 5m
       labels:
+        page: "true"
         severity: critical
     - alert: HAProxyBackendDown
       annotations:
-        description: 'HAProxy: {{ $labels.mintel_com_service }} service has been down
-          for at least 1m'
+        description: 'HAProxy: {{ $labels.mintel_com_service }} backend has been down
+          for at least 1m on all Ingress pods'
         runbook_url: https://gitlab.com/mintel/satoshi/docs/blob/master/runbooks/core/HAProxy.md#HAProxyBackendDown
       expr: sum by (mintel_com_service, label_app_mintel_com_owner) (haproxy:haproxy_backend_up:labeled)
         == 0
       for: 1m
       labels:
+        page: "false"
+        severity: critical
+    - alert: HAProxyServerInBackendUpPercentageLow
+      annotations:
+        runbook_url: https://gitlab.com/mintel/satoshi/docs/blob/master/runbooks/core/HAProxy.md#HAProxyServerInBackendUpPercentageLow
+        summary: 'HAProxy: The percentage of server up in backend for {{ $labels.mintel_com_service
+          }} service is low on ingress {{ $labels.pod }} : {{ $value }}%'
+      expr: "(\nsum by (mintel_com_service, label_app_mintel_com_owner, pod) (haproxy:haproxy_server_up:labeled)\n/\ncount
+        by (mintel_com_service, label_app_mintel_com_owner, pod) (haproxy:haproxy_server_up:labeled)\n)
+        * 100 < 75 \n"
+      for: 5m
+      labels:
+        severity: warning
+    - alert: HAProxyServerInBackendUpPercentageLow
+      annotations:
+        runbook_url: https://gitlab.com/mintel/satoshi/docs/blob/master/runbooks/core/HAProxy.md#HAProxyServerInBackendUpPercentageLow
+        summary: 'HAProxy: The percentage of server up in backend for {{ $labels.mintel_com_service
+          }} service is low on ingress {{ $labels.pod }} : {{ $value }}%'
+      expr: |
+        (
+        sum by (mintel_com_service, label_app_mintel_com_owner, pod) (haproxy:haproxy_server_up:labeled)
+        /
```

Comment:
I'm not seeing any metrics for prod for `haproxy_server_up` - so this rule will never fire. 

This could be a bigger problem (i.e. why are we not getting that metric)

Reply:
this , same as above , is a recording i wrote but is not here in the MR , i will find it

Reply:
it actually is in the MR 

https://github.com/mintel/kube-monitoring-mixins/pull/26/files?file-filters%5B%5D=.libsonnet&file-filters%5B%5D=.yaml#diff-b7121ca62a552a5fe905b031dd4a995bR103